nixos/modules/services/networking/unbound.nix

```diff
@@ -14,6 +14,18 @@ let
 
   isLocalAddress = x: substring 0 3 x == "::1" || substring 0 9 x == "127.0.0.1";
 
+  remote =
+    optionalString (cfg.enableRemoteAccess) ''
+    remote-control:
+      control-enable: ${if cfg.enableRemoteAccess then "yes" else "no"}
```

Comment:
This will always be "yes",no need to interpolate the condition

Reply:
I reflected about it while writing it but thought "better safe than sorry". In hindsight I see it only adds unnecessary computation. 
Fixed.